src/plugins/data_table/row.js

```diff
@@ -10,8 +10,16 @@ var Row = (function() {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
+        if (currentValue.bulk || ((currentValue.attr || currentValue.title) && currentValue.visible) ) {
+          return currentValue;
+        }
+      }).filter(function(currentValue) {
+         return currentValue !== undefined;
+        }
```

Comment:
Yes, the map only replaces stuff, doesn't splice. So it'd look like `[object, object, undefined, object, object, undefined]`, the filter removes the undefined parts.